packages/solutions/app-tools/src/builder/shared/builderPlugins/adapterSSR.ts

```diff
@@ -212,6 +212,7 @@ function applyFilterEntriesBySSRConfig({
   const { ssr, ssrByEntries } = serverConfig || {};
   entryNames.forEach(name => {
     if (
+      !serverConfig?.rsc &&
```

Comment:
Do we need `rscByEntries` ?

Reply:
I think we don't need it in this version and add it by user requirements later

Reply:
OK